lib/web.js

```diff
@@ -80,9 +80,7 @@ web.start = function (port, host, mailserver, user, password, basePathname) {
   }
 
   if (basePathname) {
-    io.path(basePathname + '/socket.io')
-  } else {
-    basePathname = '/'
+    io = require('socket.io')({ path: basePathname + '/socket.io' })
   }
 
   app.use(basePathname, express.static(path.join(__dirname, '../app')))
```

Comment:
Without the `basePathname = '/'` line the app will not start up without the basePathname in this line: `app.use(basePathname ...`. Will fix in a commit after this is merged 😉